boefjes/boefjes/plugins/kat_dnssec/boefje.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM noamblitz/drill:latest
+
+RUN apk add --update --no-cache python3 curl && ln -sf python3 /usr/bin/python && python3 -m ensurepip
```

Comment:
I think it would be better to use python:3.11 as base image and do `apt-get install ldnsutils` to install `drill` than continue using this ancient container with outdated and probably unpatched software.

That would also mean it isn't necessary to add the exceptions to pre-commit and pyproject.toml.

Reply:
I guess this is not a big change and `drill` is just a widely used "standard" library? Then agreed. I also considered this image as a challenge to see that we could at least easily support the alpine 3.4.0 image, but where we can upgrade the base images we can definitely upgrade them in this process.